.gitignore

```diff
@@ -41,8 +41,17 @@ docs/_build/
 docs/warnings.txt
 .DS_Store
 docs/content/examples/*
-docs/content/tutorials/models_mapping*
+docs/content/tutorials/*
+tutorials/temporary/sensitivity.zarr
```

Comment:
should we just add a blanket `*.zarr` we shouldn't have zarr files in this repo anyways?